### PR TITLE
Adds more information about collaborators

### PR DESCRIPTION
**Pull request by robalaban**
## What changes are introduced in this PR?

Adds information about collaborators by requesting on the following API:

`https://api.github.com/users/${username}`

## How can I test these changes and see any new functionality?

Pending. At the moment we rely on the standard push to S3 bucket.

## Additional information (e.g. screenshots, useful links, commands to run):

N/A

### CODE REVIEW

File: packages/backend/init-action/helpers/collaborators.js [hunk 1]
```diff
@@ -8,6 +8,32 @@ const getCollaborators = async ({ octokit, owner, repo }) => {
       core.setFailed(error.message);
     });
 
+  const collaboratorInformation = projectCollaborators.map(collaborator =>
+    octokit
+      .request(`GET /users/${collaborator}`)
+      .then(c => c.data)
```
Comment (pxlprfct) [inline packages/backend/init-action/helpers/collaborators.js]: I don't mind this sort of thing, but _maybe_ someone does?

`collaborator` maybe?

File: packages/backend/init-action/helpers/collaborators.js [hunk 2]
```diff
@@ -8,6 +8,32 @@ const getCollaborators = async ({ octokit, owner, repo }) => {
...
+      .catch(error => {
+        core.debug("collaboratorInformation");
+        core.setFailed(error.message);
+      })
+  );
+
+  (await Promise.all(collaboratorInformation)).reduce(
```
Comment (pxlprfct) [inline packages/backend/init-action/helpers/collaborators.js]: Cheeky comment maybe? Not sure

File: packages/backend/init-action/dist/index.js [hunk 3]
```diff
@@ -9051,6 +9051,9 @@ const run = async () => {
 
   const historyIndex = buildHistoryIndex(history);
 
+  core.debug("Debugging history index");
```
Comment (pxlprfct) [inline packages/backend/init-action/dist/index.js]: This can be removed?

Comment (robalaban) [reply packages/backend/init-action/dist/index.js]: Good catch, will remove.

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
Git history uploaded to https://gitchaos.s3.amazonaws.com/b9670e5087cf7bb5b1281fe1386da36c141a304f

**Comment by pxlprfct** [comment]
It might make more sense to do this from my typescript pr on the other codebase?

If you make that the base branch, you can just see the changes you made too (without all my gubbins)

Also looks like there's more eslint gubbins going on - which I _think_ is better on the ts branch

**Pull request closed without merging**